src/core/src/main/java/org/apache/jmeter/engine/DistributedRunner.java

```diff
@@ -60,7 +60,7 @@ public class DistributedRunner {
     private final int retriesDelay;
     private final int retriesNumber;
     private PrintStream stdout = new PrintStream(new SilentOutputStream());
-    private PrintStream stdErr = new PrintStream(new SilentOutputStream());
+    private PrintStream stderr = new PrintStream(new SilentOutputStream());
```

Comment:
Please revert this rename as well

Reply:
i think stderr is better, i will be consist with the brother 「stdout」 name convention